src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -123,6 +123,85 @@
     'Page',
 ]
 
+WEBPROD = 'fourfront-webprod'
+WEBPROD_TWO = 'fourfront-webprod2'
+MASTERTEST = 'fourfront-mastertest'
+HOTSEAT = 'fourfront-hotseat'
+WEBDEV = 'fourfront-webdev'
+
+
+BEANSTALK_PROD = [
```

Comment:
A singular name here surprised me, and triggered me to look at the other namings above. Because `in` works on a name, I thought `my_env in BEANSTALK_PROD` might be a string search. That's Python being overloaded badly, but it can be somewhat ameliorated by calling these `BEANSTALK_PROD_ENVS`, etc.

Reply:
I've renamed them